### PR TITLE
Permalinks and zoom

### LINKED ISSUE #538: Permalinks, not working properly after 2.0 release
This PR should resolve permalinks working for all types.

### PR DESCRIPTION
**Pull request by daRasmussen**
Fixes #538

### CODE REVIEW

File: src/featureinfo.js [hunk 1]
```diff
@@ -110,40 +109,38 @@ const Featureinfo = function Featureinfo(options = {}) {
...
-      selection.id = selectionLayer.getFeatures()[0].getId();
-      selection.type = selectionLayer.getSourceLayer().get('type');
+    const firstFeature = selectionLayer.getFeatures()[0];
+    // console.log('first Feature: ', firstFeature);
+    if (firstFeature) {
+      selection.geometryType = firstFeature.getGeometry().getType();
+      // console.log('selection geometry type: ', selection.geometryType);
+      selection.coordinates = firstFeature.getGeometry().getCoordinates();
+      // console.log('selection coordinate: ', selection.coordinates);
+      selection.id = firstFeature.getId() || firstFeature.ol_uid;
```
Comment (jokd) [inline src/featureinfo.js]: We need to allow 0 as a value:
selection.id = firstFeature.getId() != null ? firstFeature.getId() : firstFeature.ol_uid;

**Comment by markus659** [comment]
Good work fixing this issue. I agree with @jokd, I think you should remove .gitattributes, editor.json and files listed under the build and css folders, and just keep the files that pertain to the issue.

Also remove the lines with console.log. Maybe clarify the TODO comment a bit, same with the "feature is not found i maybe a cluster, therefore try again." comment.

File: src/featureinfo.js (hunk 1 above)
Comment (daRasmussen) [reply src/featureinfo.js]: @jokd sound good I will fix this.

**Comment by daRasmussen** [comment]
@markus659 sound good I will fix this.

**Pull request closed without merging**